react-first-app/src/app/components/App/Game/index.js

```diff
@@ -61,9 +56,32 @@ class Game extends React.Component {
     }
   }
 
+  jumpTo(step) {
+    this.props.dispatch(changeStep(step, (step % 2) === 0));
+  }
+
   render() {
     const history = this.state.history;
-    const current = history[this.state.stepNumber];
+    const current = history[this.props.stepNumber];
+    const winner = calculateWinner(current.squares);
```

Comment:
I think you added this lines in a merge or something. You are not using winner, moves and status